doc/guide_B2C_tenant.md

```diff
@@ -0,0 +1,188 @@
+# Guide for configuring B2C tenant for Energy Origin
+
+## Introduction
+
+This document contains a step-by-step guide on how to configure the ETT B2C production tenant in Azure.
+
+### Prerequisites
+
+The following prerequiresites must be addressed before starting the configuration.
+
+- __Existing tenant:__ A B2C instance must be available, the current instance was created by T/I by request.
+- __Invitation:__ Users must be invited to the new B2C tenant. After accepting the invitation, the user may use the switch workspace option in Azure to access the tenant.
+- __Access rights:__ Users must have sufficient access rights to configure the tenant. (Currently we use the `global administrator` role)
+
+### App registrations
+
+Create registrations for `ett-authorization-b2c` client. This client will be used by custom policies to make requests to `Authorization` service.
+
+#### B2C app registration
+
+Navigate to `App registrations`.
+
+![App registrations](images/App_registrations.png)
+
+Click `New registration`.
+
+![New app registration](images/App_registrations_new.png)
+
+Enter `ett-authorization-b2c` in `Name`. Uncheck `Grant admin consent to openid and offline_access permission` checkbox. Finally click `Register`.
+
+![New b2c app registration](images/App_registration_b2c.png)
+
+The client id of the newly generated app registration is needed to configure the Authorization sub system to accept tokens when getting consent and performing admin requests. The client id when using client-credentials outside of the custom flows, is the object id of the underlying enterprise application. Navigate to the newly created app registration, and click the mananaged app link.
+
+![New b2c app registration enterprise app](images/App_registration_b2c_enterprise_app.png)
+
+The object id will be used as subject in access tokens issued with client credentials. This id must be configured in the Authorization sub system.
+
+![New b2c app registration enterprise app oid](images/App_registration_b2c_enterprise_app_oid.png)
+
+#### Add secret
+
+Navigate to newly created app registration. Values for tenant-id and client-id can be found on this page. These values are needed later. Click `Add a certificate or secret`.
+
+![New b2c certificate](images/App_registration_b2c_certificate.png)
+
+Click `New client secret`.
+
+![New b2c certificate](images/App_registration_b2c_certificate_new.png)
+
+Leave `Description` and `Expires` fields empty. Click `Add`.  __?!? Decide on expires value__
```

Comment:
Who will make this decission? Any good practises?

Reply:
Right now we are using defaults. I would like to lower this when we have automated onboarding, so externals can renew them selfs.

Reply:
Should there be a task on it?